### PR TITLE
Add ubuntu 20 support

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by mmoll** [comment]
No idea if the remaining problems are because of https://github.com/collectd/collectd/issues/3370 or something else :man_shrugging:.

File: spec/spec_helper_methods.rb [hunk 1]
```diff
@@ -20,15 +20,15 @@ def all_supported_os_hash
     supported_os: [
```
Comment (ekohl) [inline spec/spec_helper_methods.rb:20]: This whole method appears to be unused. Can't we drop it?

File: .sync.yml [hunk 2]
```diff
@@ -1,3 +0,0 @@
----
```
Comment (ekohl) [inline .sync.yml:1]: Is this really the right thing to do?

Comment (bastelfreak) [reply .sync.yml]: a previous version of this PR did a cleanup of the old travis options. I updated the PR.

File: manifests/plugin/rabbitmq.pp [hunk 3]
```diff
@@ -61,7 +61,7 @@
 ) {
   include collectd
 
-  if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'8') >= 0 {
+  if ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8') or ($facts['os']['name'] == 'Ubuntu' and $facts['os']['release']['major'] == '20.04') {
```
Comment (ekohl) [inline manifests/plugin/rabbitmq.pp:64]: This means it no longer works for EL9 nor Fedora. While not technically supported, I think it's good to look to the future.

**Review by ekohl — COMMENTED** [review]
Should we actually have a `Boolean $collectd::is_python3_os` to remove all the duplicated logic?

**Comment by smortex** [comment]
I scratched my head with these tests too in #984 (they also now fail with old supported platforms).  A lot of these failing tests fail only due to an incompatible execution environment: each one is unique and nothing can be generalized :-/

The rspec-puppet tests in `spec/classes` / `spec/defines` already cover these aspects so I proposed to remove the problematic tests in #985.  I think that rebasing on top of it would help if you are okay with this.

File: manifests/plugin/rabbitmq.pp (hunk 3 above)
Comment (bastelfreak) [reply manifests/plugin/rabbitmq.pp]: good catch, fixed it

**Comment by bastelfreak** [comment]
> Should we actually have a `Boolean $collectd::is_python3_os` to remove all the duplicated logic?

I really like the idea but currently don't have the time to implement this :(

**Review by TuningYourCode — APPROVED** [review]

**Pull request merged**